tests/core/pyspec/eth2spec/test/helpers/block.py

```diff
@@ -35,6 +35,11 @@ def apply_randao_reveal(spec, state, block, proposer_index=None):
     block.body.randao_reveal = bls.Sign(privkey, signing_root)
 
 
+def compute_randao_mix(spec, state, randao_reveal):
```

Comment:
`merge/validator.md` should be built into the `spec` and thus this should be available as `spec.compute_randao_mix` and doesn't need ot be redefined.